python/chronos/src/bigdl/chronos/metric/forecast_metrics.py

```diff
@@ -259,3 +267,89 @@ def get_latency(func, *args, num_running=100, **kwargs):
                         "p99": round(1000 * sorted_time[int(0.99 * num_running)], 3)}
 
         return latency_list
+
+    @staticmethod
+    def plot(y,
+             std=None,
+             ground_truth=None,
+             x=None,
+             feature_index=0,
+             layout=(4,2),
+             prediction_interval=0.95,
+             figsize=(16,16),
+             output_file=None,
+             **kwargs):
+        '''
+        `Evalutor.plot` function helps users to visualize their forecasting result.
+
+        :param y: predict result, a 3-dim numpy ndarray with shape represented as
+               (batch_size, predict_length, feature_dim).
+        :param std: standard deviation, a 3-dim numpy ndarray with shape represented
+               as (batch_size, predict_length, feature_dim). Same shape as `y`.
+        :param ground_truth: ground truth, a 3-dim numpy ndarray with shape represented as
+               (batch_size, predict_length, feature_dim). Same shape as `y`.
+        :param x: input numpy array, a 3-dim numpy ndarray with shape represented
+               as (batch_size, lookback_length, feature_dim).
+        :param feature_index: a tuple/int, the instance index (along first dim) to plot.
+               Default to the first index.
+        :param layout: a 2-dim tuple, indicate the row_num and col_num to plot. Default to
```

Comment:
why the parameter layout default to be (4,2) ?

Reply:
I think this is a good layout to have a glance of the overall result, what default value you think reasonable? maybe (1,1)?

Reply:
well, I think maybe (1,1) is enough ? (Besides, the description "Default to automatically row_num and col_num." is little confusing)